src/client/src/ui/spotTile/model/spotTileModel.js

```diff
@@ -213,4 +227,16 @@ export default class SpotTileModel extends ModelBase {
         }
     }
   }
+
+  _startChartIQTimer(){
```

Comment:
given this is a hack, lets just move it to the view. And we then don't need RX, the whole rx approach seems suitable when there was data being pushed from open fin, but since that isn't the case and it's just a visual hack, can we just do it right in the view?